src/content/docs/infrastructure/host-integrations/host-integrations-list/github-integration.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Automating New Relic Service Catalog onboarding with GitHub Integration to enhance user adoption
+tags:
+  - New Relic integrations
+  - Github integration
+metaDescription: Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion.
+freshnessValidatedDate: 2024-10-29
+---
+
+
+Establishing a New Relic Service Catalog offers immense value but currently demands significant time and resource investments from users. The process involves detailed structuring of teams, user creation, assignments, and mapping of entity ownership, presenting a substantial barrier to entry. To mitigate this, we propose an automation solution that leverages data from external systems, such as GitHub, where team structures and associations are already managed. This integration aims to streamline the setup process, significantly reduce upfront efforts, and accelerate the adoption of New Relic's Service Catalog.
+
+<img
+  title="Github Integration dashboard"
+  alt="Github Integration dashboard"
+  src="/images/github-integration-screenshot.webp"
+/>
+
+<figcaption>
+After setting up our Github integration, see your data of teams, repos and users right out of the box.
+</figcaption>
+
+## Prerequisites [#prerequisites]
+
+Users need to ensure that they are logged in to their Github accounts. This needs to be the same github account in which they want to install the github app that New Relic will be providing during installation step.
```

Comment:
GitHub is always GitHub. And never Github or github. The casing is inconsistent across the topic. Can we please correct this?